ui/src/services/experiment/ExperimentStatus.js

```diff
@@ -13,7 +13,7 @@ export const getExperimentStatus = (experiment) => {
   const statusMapping = {
     Deactivated: {
       label: "Deactivated",
-      color: "subdued",
+      color: "#6A717D",
```

Comment:
Fixed this colour which is no longer supported as part of the EUI colour palette; it now needs to be specified manually with a colour hex.